routes/campaign.php

```diff
@@ -186,6 +186,7 @@
 // Timeline Elements
 Route::post('/editing/timeline-elements/{timeline_element}/confirm-editing', 'EditingController@confirmTimelineElement')->name('timeline-elements.confirm-editing');
 Route::post('/editing/timeline-elements/{timeline_element}/keep-alive', 'EditingController@keepAliveTimelineElement')->name('timeline-elements.keep-alive');
+Route::get('/timeline/{timeline}/era/{era}/list', 'Timelines\TimelineEraController@positionList')->name('timelines.era-list');
```

Comment:
The model is called timeline_era. Does the model get bound properly with timeline_era?